### PR TITLE
make polyvox entities visible again

### PR DESCRIPTION
**Pull request by sethalves**
- make polyvox entities visible again

### CODE REVIEW

**Comment by sethalves** [comment]
to test:
- go to hifi://eschatology/99.143,1.50435,86.0381/0,0.962793,0,0.270242
- make sure you see the PolyVox entities (the lump green ground)
https://hifi-metaverse.s3-us-west-1.amazonaws.com/snapshots/images/hifi-snap-original-26843.jpg

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/6926/app_12341_PR.apk

File: libraries/entities-renderer/src/RenderablePolyVoxEntityItem.h [hunk 1]
```diff
@@ -165,7 +165,7 @@ class PolyVoxEntityRenderer : public TypedEntityRenderer<RenderablePolyVoxEntity
     PolyVoxEntityRenderer(const EntityItemPointer& entity);
     
 protected:
-    virtual ItemKey getKey() override { return ItemKey::Builder::opaqueShape(); }
+    virtual ItemKey getKey() override { return ItemKey::Builder::opaqueShape().withTagBits(render::ItemKey::TAG_BITS_0 | render::ItemKey::TAG_BITS_1); }
```
Comment (ZappoMan) [inline libraries/entities-renderer/src/RenderablePolyVoxEntityItem.h]: ?????what?

Comment (sethalves) [reply libraries/entities-renderer/src/RenderablePolyVoxEntityItem.h]: The tag bits are being used to determine which views an entity appears in.  We changed everything except PolyVox, which ended up invisible in the main view, the mirrors, etc.  This fixes that.

These changes were made in support of being able to see worn entities in a mirror without having them obscure your 1st-person view.

Comment (sethalves) [reply libraries/entities-renderer/src/RenderablePolyVoxEntityItem.h]: This change should have been a part of this PR:  https://github.com/highfidelity/hifi/pull/12158

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/6926/HighFidelity-Beta-PR12341-33ca1fd3448dfc9e3e4ef0f89cb6d978cd825c88.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/6926/HighFidelity-Beta-PR12341-33ca1fd3448dfc9e3e4ef0f89cb6d978cd825c88.exe

**Comment by QAadahlin** [comment]
Currently Testing

**Comment by QAadahlin** [comment]
@conklin94122 Completed testing for PR#12341. PASSED.

**Pull request merged**